Docsite-HTML-Files/Avatar.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar Component</title>
+    <script
```

Comment:
you can instead use the FontAwesome CDN and import it directly into your CSS file
Like you have done for Google Fonts